etna/lib/etna/application.rb

```diff
@@ -59,6 +60,21 @@ def configure(opts)
     end
   end
 
+  def setup_yabeda
+    Yabeda.configure!
+  end
+
+  def write_job_metrics(name)
```

Comment:
Jobs, such as one offs or scheduled processes, don't have a webserver target for prometheus to pull data from.  The fix for this is simply to use the textfile collector of the node exporter (https://github.com/prometheus/node_exporter#textfile-collector) to find files saved by jobs that contain metrics to be exported to prometheus.  Jobs simply need call this function with a unique name for that job to export their metrics via the node_collector on its host.  Example given below in the prometheus/commands.rb